lib/wos_queries.rb

```diff
@@ -263,9 +258,11 @@ def search_by_doi_params(doi)
     end
 
     # @param name [String] a CSV name pattern: {last name}, {first_name} [{middle_name} | {middle initial}]
+    # @param institutions [Array<String>] a set of institutions the author belongs to
     # @return [Hash] search query parameters
-    def search_by_name_params(name)
-      user_query = "AU=(#{name_query(name)}) AND AD=(#{institutions.join(' OR ')})"
+    def search_by_name_params(name, institutions)
+      user_query = "AU=(#{name_query(name)})"
+      user_query += " AND AD=(#{institutions.join(' OR ')})" unless institutions.empty?
```

Comment:
Previously the default was Stanford, now the default is no constraint.  Right?  This changes the behavior of the task `wos_publications_for_name` in `sw.rake`.

Reply:
Oh, if this class is already used in a rake task, I'll have to change that rake task in this PR.  Thanks, I'll do that and look for any other uses of it (probably none yet).

Reply:
Done.